server/peachworks.js

```diff
@@ -1,70 +1,75 @@
-const fetch = require('node-fetch');
-const queryString = require('query-string');
+const fetch = require("node-fetch");
+const queryString = require("query-string");
 
 // for every URL path that starts with /api/, send request to upstream API service
 const peachworksApiUrl = (url, otherQueries = {}) => {
   const peachworksAccountId = process.env.PEACHWORKS_ACCOUNT_ID;
   const peachworksAccessToken = process.env.PEACHWORKS_ACCESS_TOKEN;
 
   let resultQuery = Object.assign({}, otherQueries, {
-    "access_token": peachworksAccessToken
+    access_token: peachworksAccessToken
```

Comment:
I wanted to make this one a string to make it obvious it needs to remain underscored instead of camel case for the api to work. does it break the linting?